### DISCUSSION TITLE
Please come to tell what is your favorite song!! :D

### DISCUSSION THREAD

**Discussion by vittunyuta-aom** [NAIST-SE/AttractingContributors#45] [General]
please add your favorite below the topic 'list of favorite song' in README file in this repository https://github.com/aommoaGitHub/SuperFavoriteSongs

**Comment by tattiyaS** [comment]
Hope you like Taylor Swift!

**Comment by nosfill** [comment]
I added!

**Comment by 278Mt** [comment]
added!
and plz help my favorite movies!
https://github.com/278mt/AttractingContributors
thx;-)

**Comment by manzoku23** [comment]
Added my favorite one.
Please help me
https://github.com/Exe-dev/AttractingContributors-POKEMON
